modules/oadp-using-data-mover-for-csi-snapshots.adoc

```diff
@@ -139,14 +128,12 @@ spec:
         - openshift
         - aws
         - csi
-        - vsm <5>
+        - vsm
 ----
-<1> OADP 1.2 only. Optional: Specify the upper limit of the number of snapshots allowed to be queued for backup. The default value is 10.
-<2> OADP 1.2 only. Optional: Specify the upper limit of the number of snapshots allowed to be queued for restore. The default value is 10.
-<3> OADP 1.2 only. Optional: Specify the number of days, between running Restic pruning on the repository. The prune operation repacks the data to free space, but it can also generate significant I/O traffic as a part of the process. Setting this option allows a trade-off between storage consumption, from no longer referenced data, and access costs.
-<4> OADP 1.2 only. Optional: Specify VolumeSync volume options for backup and restore.
-<5> OADP 1.2 only.
-
+<1> Optional: Specify the upper limit of the number of snapshots allowed to be queued for backup. The default value is 10.
+<2> Optional: Specify the upper limit of the number of snapshots allowed to be queued for restore. The default value is 10.
+<3> Optional: Specify the number of days, between running Restic pruning on the repository. The prune operation repacks the data to free space, but it can also generate significant I/O traffic as a part of the process. Setting this option allows a trade-off between storage consumption, from no longer referenced data, and access costs.
```

Comment:
Same suggestion as above:
```suggestion
<3> Optional: Specify the number of days between running Restic pruning on the repository. The prune operation repacks the data to free space, but it can also generate significant I/O traffic as a part of the process. Setting this option allows a trade-off between storage consumption, from no longer referenced data, and access costs.
```